origami.json

```diff
@@ -1,5 +1,6 @@
 {
-	"description": "Accessible, content-aware widget for expanding and collapsing content",
+	"description": "This is an accessible and content-aware component to aid products in expanding and collapsing content.",
```

Comment:
I know you're including "Accessible" from the previous description, but I don't think we should. Technically we build all our components to be accessible and this should just be standard messaging on the site rather than highlighted in an individual component I think